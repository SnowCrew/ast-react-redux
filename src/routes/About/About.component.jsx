const About = () => {
  return <div>Information about shop</div>;
};

export default About;
