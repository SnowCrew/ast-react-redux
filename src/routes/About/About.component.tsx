import "./About.styles.css";

const About = () => {
  return (
    <div className="about">
      <h3>Get tired of Lorem ipsum data?!</h3>
      <h3>Didn't you find any free e-commerce API?!</h3>
      <p>
        Get tired of Lorem ipsum data?! Didn't you find any free e-commerce
        API?! fakeStoreApi is a free online REST API that you can use whenever
        you need Pseudo-real data for your e-commerce or shopping website
        without running any server-side code. It's awesome for teaching
        purposes, sample codes, tests, etc.
      </p>
    </div>
  );
};

export default About;
