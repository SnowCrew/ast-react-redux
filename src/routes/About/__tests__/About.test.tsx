import { render } from "@testing-library/react";
import About from "../About.component";

describe("About render without errors", () => {
  it("About rendering", () => {
    expect(render(<About />)).toMatchSnapshot();
  });
});
