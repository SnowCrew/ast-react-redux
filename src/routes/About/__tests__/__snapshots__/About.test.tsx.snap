// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About render without errors About rendering 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="about"
      >
        <h3>
          Get tired of Lorem ipsum data?!
        </h3>
        <h3>
          Didn't you find any free e-commerce API?!
        </h3>
        <p>
          Get tired of Lorem ipsum data?! Didn't you find any free e-commerce API?! fakeStoreApi is a free online REST API that you can use whenever you need Pseudo-real data for your e-commerce or shopping website without running any server-side code. It's awesome for teaching purposes, sample codes, tests, etc.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="about"
    >
      <h3>
        Get tired of Lorem ipsum data?!
      </h3>
      <h3>
        Didn't you find any free e-commerce API?!
      </h3>
      <p>
        Get tired of Lorem ipsum data?! Didn't you find any free e-commerce API?! fakeStoreApi is a free online REST API that you can use whenever you need Pseudo-real data for your e-commerce or shopping website without running any server-side code. It's awesome for teaching purposes, sample codes, tests, etc.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
